packages/bolt-connection/src/bolt/bolt-protocol-v1.js

```diff
@@ -454,6 +455,24 @@ export default class BoltProtocol {
     return observer
   }
 
+  /**
+   * Send a TELEMETRY through the underlying connection.
+   *
+   * @param {object} param0 Message params
+   * @param {number} param0.api The API called
+   * @param {object} param1 Configuration and callbacks callbacks
+   * @param {function()} param1.onCompleted Called when completed
+   * @param {function()} param1.onError Called when error
+   * @return {StreamObserver} the stream observer that monitors the corresponding server response.
+   */
+  telemetry ({ api }, { onError, onCompleted } = {}) {
+    const observer = new CompletedObserver()
+    if (onCompleted) {
+      onCompleted()
```

Comment:
I don't think it can cause any harm to the existing code to not call the onComplete, but it is a quite surprising behaviour the request never finished. So, for keeping the same external of the version which actual send the message, I prefer to call the onCompleted